src/python/gudhi/representations/vector_methods.py

```diff
@@ -912,28 +918,29 @@ def fit(self, X, y=None):
 
     def transform(self, X):
         """
-        Compute the persistence landscape for each persistence diagram individually and concatenate the results.
+        Compute the persistence lengths for each persistence diagram individually and concatenate the results.
```

Comment:
When I read "concatenate", I expect the output to be a 1d array of length N*k.

Reply:
I removed the "concatenate" on [3bbd3af](https://github.com/GUDHI/gudhi-devel/pull/1117/commits/3bbd3af834902e79e83b990ae246dc9f1469e2d7)